### ISSUE TITLE
Comment system

### ISSUE THREAD

**Issue by AllanJeremy** [AllanJeremy/esomo-upgrade#24]
**Teacher, student, principal feature**

- [ ] 1. Teachers can comment on assignments and assignment submissions
- [ ] 2. Students can comment on assignments* (if teacher allowed comments) as well as assignment submissions
- [ ] 3. Principal can comment on schedules
- [ ] 4. Teachers can comment on schedules

This feature should ideally be implemented as soon as all sections requiring comments are done so that every section can be developed and tested at once

**Comment by mwauramuchiri** [comment]
Comments on schedules?

**Comment by AllanJeremy** [comment]
Yes comments on schedules. Communication between teachers and principal through schedules.

**Comment by mwauramuchiri** [comment]
I think this is done?
